oryx-app-mllib/src/main/java/com/cloudera/oryx/app/mllib/kmeans/KMeansUpdate.java

```diff
@@ -154,28 +150,30 @@ private PMML kMeansModelToPMML(KMeansModel model, Map<Integer,Long> clusterSizes
     return pmml;
   }
 
-  private ClusteringModel pmmlClusteringModel(KMeansModel model, Map<Integer,Long> clusterSizesMap) {
-    ClusteringModel clusteringModel =
-        new ClusteringModel(AppPMMLUtils.buildMiningSchema(inputSchema),
-            new ComparisonMeasure(ComparisonMeasure.Kind.DISTANCE)
-                .withMeasure(new SquaredEuclidean()),
-            MiningFunctionType.CLUSTERING,
-            ClusteringModel.ModelClass.CENTER_BASED,
-            model.clusterCenters().length)
-            .withNumberOfClusters(model.k());
-
-    if (initializationStrategy.equals(KMeans.K_MEANS_PARALLEL())) {
```

Comment:
I think this can be removed as it's not really the algo name, and isn't standardized or used by tools